indico/web/client/styles/modules/_admin.scss

```diff
@@ -28,7 +28,6 @@
 }
 
 table.ip-network-group .actions {
```

Comment:
Why is that class even used on the "News" page? 🙈

Reply:
no idea, but it works so I didn't want to touch it :smile: